### PR TITLE
Add Some Attributes Support

### PR DESCRIPTION
**Pull request by AliChraghi**

### CODE REVIEW

File: syntaxes/v.tmLanguage.json [hunk 1]
```diff
@@ -294,7 +294,7 @@
 						},
 						{
 							"name": "keyword.control.v",
-							"match": "\\b(as|in|is|or|break|continue|default|unsafe|match|if|else|for|go|goto|defer|return|shared|select|rlock|lock|atomic|asm)\\b(?=\\s*\\()"
+							"match": "\\b(as|in|is|or|break|continue|default|unsafe|match|if|else|for|go|goto|defer|return|shared||select|rlock|lock|atomic|asm)\\b(?=\\s*\\()"
```
Comment (danieldaeschle) [inline syntaxes/v.tmLanguage.json:297]: seems like a mistake

File: syntaxes/v.tmLanguage.json [hunk 2]
```diff
@@ -310,7 +310,7 @@
 							"name": "meta.expr.numeric.cast.v"
 						},
 						{
-							"match": "(bool|byte|byteptr|charptr|voidptr|string|ustring|rune)(?=\\s*\\()",
+							"match": "(bool|byte|byteptr|charptr|voidptr|string|ustring|rune|any)(?=\\s*\\()",
```
Comment (danieldaeschle) [inline syntaxes/v.tmLanguage.json:313]: any is not a builtin type

**Review by danieldaeschle — CHANGES_REQUESTED** [review]

File: syntaxes/v.tmLanguage.json (hunk 2 above)
Comment (AliChraghi) [reply syntaxes/v.tmLanguage.json:313]: thanks fixed!

File: .gitignore [hunk 3]
```diff
@@ -1,6 +1,7 @@
 .vscode/ipch
 *.code-workspace
 node_modules
+package-lock.json
```
Comment (serkonda7) [inline .gitignore:4]: As far as I remember, the lockfile should be tracked by git

**Review by serkonda7 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 3 above)
Comment (AliChraghi) [reply .gitignore:4]: fixed

Comment (serkonda7) [reply .gitignore:4]: and remove it from the gitignore please

Comment (AliChraghi) [reply .gitignore:4]: I added the package.json file 😂

Comment (AliChraghi) [reply .gitignore]: refixed

Comment (AliChraghi) [reply .gitignore]: please don't merge now!
I should to add a small thing

**Review by danieldaeschle — CHANGES_REQUESTED** [review]
Fix the last comment

**Comment by AliChraghi** [comment]
> Fix the last comment

it's fixed, But I think two things need to be fixed
1:
```v
a << b       // << not detected as a operator in vscode
```
2: i will be fixed **1** but, But the color of each variable is equal to the color of the operators

**Comment by danieldaeschle** [comment]
Your have `||`, i think that's a mistake.

**Comment by AliChraghi** [comment]
I fixed all of these in several commits but github is just showing the first commit

**Review by serkonda7 — APPROVED** [review]
As far as I can tell, this PR is fine now.

**Review by danieldaeschle — APPROVED** [review]

**Comment by AliChraghi** [comment]
> As far as I can tell, this PR is fine now.

thanks, so i create another PR for second commits

**Pull request merged**